ios/brave-ios/App/iOS/Delegates/AppState.swift

```diff
@@ -190,12 +190,6 @@ public class AppState {
     }
     switches.append(.init(key: .rewardsFlags, value: BraveRewards.Configuration.current().flags))
 
-    if !AppConstants.buildChannel.isPublic {
-      switches.append(.init(key: .init(rawValue: "env-leo"), value: "staging"))
-      switches.append(.init(key: .init(rawValue: "env-ai-chat.bsg"), value: "dev"))
-      switches.append(.init(key: .init(rawValue: "env-ai-chat-premium.bsg"), value: "dev"))
-    }
```

Comment:
Use `#if !IS_OFFICIAL_BUILD` instead of deleting

Reply:
Sorry its `OFFICIAL_BUILD` (not `IS_OFFICIAL_BUILD`)

so

```swift
#if !OFFICIAL_BUILD
      switches.append(.init(key: .init(rawValue: "env-leo"), value: "staging"))
      switches.append(.init(key: .init(rawValue: "env-ai-chat.bsg"), value: "dev"))
      switches.append(.init(key: .init(rawValue: "env-ai-chat-premium.bsg"), value: "dev"))
#endif
```

Reply:
Realized last night that `OFFICIAL_BUILD` is not available in Release yet. I can either uplift if you'd like or you can just use `#if DEBUG` instead here

Reply:
It will still work without this once your `.env` is setup correctly. That's why I deleted it. This was for when we didn't have the `.env` stuff ready, so it's not needed :)